schemas/schema.graphql

```diff
@@ -20,6 +21,7 @@ type Query {
   channels: [Channel!]!
   friends: [User!]!
   galleries(userId: String!): [Gallery!]!
+  
```

Comment:
I think you can remove the extra line here.

Reply:
ok!